packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/trust-store-revocation.ts

```diff
@@ -0,0 +1,80 @@
+import { Construct } from 'constructs';
+import { ITrustStore } from './trust-store';
+import { IBucket } from '../../../aws-s3';
+import { Resource } from '../../../core';
+import { CfnTrustStoreRevocation } from '../elasticloadbalancingv2.generated';
+
+/**
+ * properties for the trust store revocation
+ */
+export interface TrustStoreRevocationProps {
+
+  /**
+   * The trust store
+   */
+  readonly trustStore: ITrustStore;
+
+  /**
+   * The revocation file to add
+   */
+  readonly revocationContents: RevocationContent[];
+}
+
+/**
+ * Information about a revocation file
+ */
+export interface RevocationContent {
+  /**
+   * The type of revocation file
+   *
+   * @default RevocationType.CRL
+   */
+  readonly revocationType?: RevocationType;
+
+  /**
+   * The Amazon S3 bucket for the revocation file
+   */
+  readonly bucket: IBucket;
```

Comment:
In [CFn document](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-elasticloadbalancingv2-truststorerevocation-revocationcontent.html), this property is optional.
However, deploying without setting it resulted in an error, so I made it required.

Also, the documentation states the following:
> You must specify S3Bucket and S3Key.